walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -91,42 +92,43 @@
 
   /**
    * Recursive helper function used to find all combinations of size r of the input array arr[].
-   * chosen[]: temporary array to store current combination.
-   * start & end: staring and ending indexes in arr[].
-   * index: current index in data[].
+   * @param arr[] The input array.
+   * @param chosen[] Temporary array to store current combination.
+   * @param start Starting index in arr[].
+   * @param end Ending index in arr[].
+   * @param index Current index in chosen[].
+   * @param r The size of combinations to be found.
    */
   private void combinationUtil(String arr[], String chosen[], int start, int end, int index, int r) { 
-    // If current combination is ready to be added, then add it.
     if (index == r) {
-      optionalAttendeesCombs.add(Arrays.asList(chosen));
+      // Combination is ready to be added, add it to the list.
+      this.optionalAttendeesCombinations.add(Arrays.asList(chosen));
       return;
     }
 
     // Replace index with all possible elements. 
     // The condition "end-i+1 >= r-index" makes sure that including one element
     // at index will make a combination with remaining elements at remaining positions.
```

Comment:
is this line wrapping right?

Reply:
I'm confused. What is supposed to be the correct line wrapping?